.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
-build/
+# build/
```

Comment:
Can you clarify why we need to include the `build` folder?

Reply:
I think what we need is a `.npmignore` file which excludes all files except `build`. For git, we want to ignore `build`

Reply:
For testing, we need it until we get the CI and npm package running. In the `init` phase, it installs this branch. Before we merge, I need to clean this part up. I'll add the `.npmignore`.